server/graphql/v2/object/Host.js

```diff
@@ -698,7 +698,10 @@ export const Host = new GraphQLObjectType({
             `
             SELECT o.id FROM "Orders" o
             JOIN "Transactions" t ON t."OrderId" = o.id
-            WHERE t."HostCollectiveId" = :hostCollectiveId AND o.status = 'DISPUTED'
+            WHERE t."HostCollectiveId" = :hostCollectiveId
```

Comment:
This one was [taking](https://sentry.io/organizations/open-collective/performance/oc-api:1237505d7dbe48048173596c16222b7c/?project=5199682&query=transaction.duration%3A%3C15m+http.method%3APOST&statsPeriod=24h&transaction=GraphQL%3A+ProcessExpense#span-8429a39a841ceb52) almost 800ms in prod.

Cost without deleted:
> Limit  (cost=0.09..45615.94 rows=1 width=4) (actual time=566.355..566.357 rows=0 loops=1)

With deleted:
> Limit  (cost=0.17..64.95 rows=1 width=4) (actual time=0.035..0.036 rows=0 loops=1)

Reply:
Also, the sequelize version of this would have been much simpler (sequelize automatically filters deleted entries):

```es6
return Boolean(
  await models.Order.findOne({
    where: { status: 'DISPUTED' },
    include: [{ model: models.Transaction, where: { HostCollectiveId: host.id } }],
  }),
);
```

Reply:
Yep, would rather have it as Sequelize JS here.